app/js/app/directives/react_classes/QuestionBlock.js

```diff
@@ -450,7 +452,7 @@ define(["react", "jquery"], function(React,$) {
 											<label><input type="checkbox" checked={this.props.doc.randomiseChoices} onChange={this.onCheckboxChange.bind(this, "randomiseChoices")} />Randomise Choices</label>
 										</div>
 									</div>
-									<div className="row" style={{display: this.props.doc.type == "isaacStringMatchQuestion" ? "block" : "none"}}>
+									<div className="row" style={{display: this.props.doc.type in ["isaacStringMatchQuestion", "isaacRegexMatchQuestion"] ? "block" : "none"}}>
```

Comment:
To be consistent with the changes on 316 above, this could maybe be `["isaacStringMatchQuestion", "isaacRegexMatchQuestion"].includes(this.props.doc.type) ? "block" : "none"`

Reply:
If it's okay I'll leave it as is? I find that it's more readable this way (just my preference)

Reply:
Is this ES6, though? Will it work in Safari? We don't do any transpilation or anything clever, so if it's not valid ES5, it won't work in some browsers.

Reply:
I'll change it - I can't determine whether `[value] in [list]` is valid or not :/